### PR TITLE
zh-ch: translation /js/guide/index.md

### PR DESCRIPTION
**Pull request by PeterJaq**
translate js\guide\index.md

### CODE REVIEW

File: site/zh-cn/js/guide/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+这篇教程将会涵盖一下章节：
+
+* [Tensors(张量)和Operations(操作)](tensors_operations.md) — 此章节将介绍Tensorflow.js的组成部分，如张量，数据，形状以及数据类型等
+* [Platform(平台) & Environment(环境)](platform_environment.md) — 此章节为Tensorflow.js在不同平台以及环境概述，并简述了Tensorflow.js在不同平台上的取舍。
+* [Models(模型) and Layers(层)](models_and_layers.md) — 此章节介绍了如何利用Layers以及Core的API在Tensorflow.js中建立模型。
+* [训练模型](train_models.md) — 此章节介绍了如何训练：模型，优化器，损失函数，评价标准以及变量。
```
Comment (kuri-leo) [inline site/zh-cn/js/guide/index.md]:
```suggestion
* [训练模型](train_models.md) — 此章节介绍了如何训练：models（模型），optimizers（优化器），losses（损失函数），metrics（评价标准）以及variables（变量）。
```

File: site/zh-cn/js/guide/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+* [保存和加载模型](save_load.md) — 此章节将会指导你如何在Tensorflow.js中保存以及加载模型
+* [模型转换](conversion.md) — 此章节主要介绍在Tensorflow.js中可用的模型类型以及其转换细节
+* [与Python tf.keras的不通](layers_for_keras_users.md) — 此章节将会阐述tensorflow.js和python'tf.keras'的主要区别和功能，并了解JavaScript中的API
```
Comment (kuri-leo) [inline site/zh-cn/js/guide/index.md]:
```suggestion
* [与Python tf.keras的差异](layers_for_keras_users.md) — 此章节将会阐述tensorflow.js和python'tf.keras'主要的区别和功能，并了解JavaScript中的API规范。
```

File: site/zh-cn/js/guide/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+这篇教程将会涵盖一下章节：
+
+* [Tensors(张量)和Operations(操作)](tensors_operations.md) — 此章节将介绍Tensorflow.js的组成部分，如张量，数据，形状以及数据类型等
```
Comment (kuri-leo) [inline site/zh-cn/js/guide/index.md]:
```suggestion
* [Tensors(张量)和Operations(操作)](tensors_operations.md) — 此章节将介绍Tensorflow.js的组成部分，如张量，数据，形状以及数据类型等。
```

**Comment by kuri-leo** [comment]
LGTM

**Comment by kuri-leo** [comment]
Hi @lamberta, it seems that my review commits caused a CLA issue with the bot and it requires a manually update. So could you please check this? 

Thanks in advance,

> So there's good news and bad news.
> 
> 👍 The good news is that everyone that needs to sign a CLA (the pull request submitter and all commit authors) have done so. Everything is all good there.
> 
> 😕 The bad news is that it appears that one or more commits were authored or co-authored by someone other than the pull request submitter. We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that here in the pull request.
> 
> _Note to project maintainer: This is a terminal state, meaning the `cla/google` commit status will not change from this state. It's up to you to confirm consent of all the commit author(s), set the `cla` label to `yes` (if enabled on your project), and then merge this pull request when appropriate._
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdocs%2Fpull%2F587) for more info**.
